.changeset/smart-chefs-fail.md

```diff
@@ -0,0 +1,5 @@
+---
+'@primer/react': minor
+---
+
+IconButton: introduce tooltip behavior behind the prop `_enableTooltip_`
```

Comment:
Note: We are still discussing the prop name as whether to go with a regular prop name versus  with an underscore type of identifier to make it differentiate from being a regular prop (you can read more about it [here](https://github.com/github/primer/discussions/3027#discussioncomment-8387644))